packages/web-runtime/package.json

```diff
@@ -25,7 +25,7 @@
     "marked": "^4.0.12",
     "oidc-client": "1.11.5",
     "owncloud-design-system": "^14.0.0-alpha.2",
-    "owncloud-sdk": "~3.0.0-alpha.14",
+    "owncloud-sdk": "owncloud/owncloud-sdk#experimental-next",
```

Comment:
Sorry to be annoying but "would need a changelog item" ✍🏽

Reply:
this is just for experimental cases ;)